### PR TITLE
[develop] Carlos - implement archive list scenario

### LINKED ISSUE #57: Implement Archive list Scenario
- [x] Implement scenario steps
- [x] Implement steps definition
- [x] Implement the necessary classes

### PR DESCRIPTION
**Pull request by carlosmc23**

### CODE REVIEW

File: src/test/java/org/fundacionjala/trello/pages/board/ArchivedItemsBoard.java [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+     */
+    @Override
+    public boolean isDisplayed() {
+        action.waitForVisibility(tittle);
+        return tittle.isDisplayed();
+    }
+
+    public ArchivedItemsBoard switchItems() {
+        action.waitForVisibility(switchButton);
+        if (switchButton.getText().contains("list")) {
```
Comment (luqp) [inline src/test/java/org/fundacionjala/trello/pages/board/ArchivedItemsBoard.java]: "list" should be a constant

File: src/test/resources/features/lists/Update_list.feature [hunk 2]
```diff
@@ -18,3 +18,11 @@ Feature: List
       | name | reviewed |
     Then I should have a list updated with:
       | name | reviewed |
+
+  @deleteBoard
```
Comment (luqp) [inline src/test/resources/features/lists/Update_list.feature]: Change it by cleanData

**Review by luqp — COMMENTED** [review]
Hi, I leave some comments, please take a look.
When I run only this `"Archive a list"` test scenario the test is pass,
but when I run it with all tests with Runner class I get this error:  
`java.lang.AssertionError: expected [true] but found [false]`

File: src/test/java/org/fundacionjala/trello/pages/board/ArchivedItemsBoard.java (hunk 1 above)
Comment (carlosmc23) [reply src/test/java/org/fundacionjala/trello/pages/board/ArchivedItemsBoard.java]: done, thanks

**Review by Snip-77 — APPROVED** [review]

**Pull request merged**